node/connection_base.js

```diff
@@ -71,6 +73,31 @@ function TChannelConnectionBase(channel, direction, remoteAddr) {
 
     self.tracer = self.channel.tracer;
     self.ops.startTimeoutTimer();
+
+    if (direction === 'out') {
+        self.channel.connectionsInitiatedStat.increment(1, {
+            'host-port': self.channel.hostPort || '0.0.0.0:0',
+            'peer-host-port': self.remoteAddr
+        });
+    } else {
+        self.channel.connectionsAcceptedStat.increment(1, {
+            'host-port': self.channel.hostPort,
+            'peer-host-port': self.remoteAddr
+        });
+    }
+
+    if (self.reportInterval > 0) {
+        self.reportTimer = self.timers.setTimeout(
+            onReport, self.reportInterval
+        );
+    }
+
+    function onReport() {
+        self.channel.connectionsActiveStat.update(1, {
```

Comment:
it is "The number of currently active outbound connections from the process to the peer.", so it is per peer bases. I can move the connection to peer level. It cannot be at the channel level by definition.